electron/src/menu/TrayHandler.test.main.ts

```diff
@@ -32,11 +32,11 @@ describe('initTray', () => {
     const tray = new TrayHandler();
     tray.initTray(TrayMock);
 
-    assert.strictEqual(Object.keys(tray['icons']!).length, 3);
-    assert.strictEqual(tray['icons']!.badge.constructor.name, 'NativeImage');
-    assert.strictEqual(tray['icons']!.tray.constructor.name, 'NativeImage');
-    assert.strictEqual(tray['icons']!.trayWithBadge.constructor.name, 'NativeImage');
-    sinon.assert.match(tray['trayIcon']!, sinon.match.defined);
+    assert.strictEqual(Object.keys((tray as any).icons!).length, 3);
+    assert.strictEqual((tray as any).icons!.badge.constructor.name, 'NativeImage');
+    assert.strictEqual((tray as any).icons!.tray.constructor.name, 'NativeImage');
+    assert.strictEqual((tray as any).icons!.trayWithBadge.constructor.name, 'NativeImage');
+    sinon.assert.match((tray as any).trayIcon!, sinon.match.defined);
```

Comment:
Can we get rid of `any` here and stick to the old format?

Reply:
Maybe we should remove the this rule then: https://github.com/wireapp/wire-web-packages/blob/master/packages/eslint-config/eslintrc.json#L19 (`dot-notation`)

Reply:
/cc @bennyn @AndyLnd? What do you guys think?